go/src/api/project.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
I don't think this one would still be needed, right? There's also a root-level `project.json` with similar targets, which in a "standalone" setup would probably be the place where you define these tasks